handgesture/README.md

```diff
@@ -1,21 +1,28 @@
 # HandView
 
-### HandView attributes
+## Gradle Import
 
-**animationType**: `moveUp`, `moveDown`, `moveLeft`, `moveRight`, `singleTouch`, `doubleTouch`, `pressAndHold`, `translation`
-    
-**hideOnTouch**: Hide view when touching screen. Default: `true`
-
-**repeatAnimation**: Repeat animation indefinitely. Default: `true` 
+To import the library, you first need to add our repository in app/build.gradle:
 
-**animationDelay**: Animation delay in seconds. Default: 1 second
-
-**translateX** (only for ```animationType="translation"```): Translation in X coordinate from initial hand position.
+```
+repositories {
+    mavenLocal()
+    maven {
+        url "http://archiva.educativo.eu:8081/repository/internal/"
+    }
+}
+```
 
-**translatey** (only for ```animationType="translation"```): Translation in Y coordinate from initial hand position. 
+Then, add the following dependency:
 
-**duration** (only for ```animationType="translation"```): Duration of hand translation in seconds. Default: 1 second.
+```
+dependencies {
+   ...
+   compile 'org.literacyapp.startguide:1.0.3'
```

Comment:
@gscdev Change to `org.literacyapp.startguide:handgesture:1.0.0` (to separate it from the main module of the app).

We'll generate an `.aar` file for this module, and upload it to our Maven repository.